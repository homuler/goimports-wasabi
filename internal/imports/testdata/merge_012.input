// merge import declarations with comments
package main

// doc comment (1)
import (
	// doc comment for fmt
	"fmt"
	"github.com/homuler/foo"
)

// doc comment for context
import "context"
// doc comment for baz
import "local/baz"
// doc comment (2)
import (
	"local/qux" // comment for qux
	// doc comment for bar
	"github.com/homuler/bar"
)

var (
	x = foo.X
	y = bar.Y
	w = baz.W
	z = qux.Z
)

func main() {
	fmt.Println(context.Background())
	fmt.Printf("%v %v %v %v\n", x, y, w, z)
}
