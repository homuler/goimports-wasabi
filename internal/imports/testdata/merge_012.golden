// merge import declarations with comments
package main

// doc comment (1)
// doc comment (2)
import (
	// doc comment for context
	"context"
	// doc comment for fmt
	"fmt"

	// doc comment for bar
	"github.com/homuler/bar"
	"github.com/homuler/foo"

	// doc comment for baz
	"local/baz"
	"local/qux" // comment for qux
)

var (
	x = foo.X
	y = bar.Y
	w = baz.W
	z = qux.Z
)

func main() {
	fmt.Println(context.Background())
	fmt.Printf("%v %v %v %v\n", x, y, w, z)
}
